### ISSUE TITLE
How to learn the layer-wise similarity scores?

### ISSUE THREAD

**Issue by cdb342** [mshahbazi72/cGANTransfer#1]

**Comment by mshahbazi72** [comment]
> _No description provided._

Could you please provide a description of the question you have?

Each layer has its own set of learnable weights used for linear combining the layer-wise batch normalization parameters (shift and scale) of the pretraining classes. These source parameters are also further optimized by the target training data while learning the combination weights.

**Comment by cdb342** [comment]
Thanks
